free-2.md

```diff
@@ -11,34 +12,120 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The users I have have in mind are the users of latex who find it's syntax
+somewhat obtrusive. Right now, when people use LaTeX they need to type a lot of
+things, and sometimes the semantics are not fully clear.
+
+For example, to use an environment you have to do:
+
+```
+\begin{center}
+  <stuff to center>
+\end{center}
+```
+
+Notably many LaTeX users end up indenting the contents of an environment
+even though the language doesn't require it. Even those that choose not to
+agree that doing so make the code more readable.
+
+I would expand/restrict LaTeX to make it a whitespace sensitive language, which
+is just as full-featured as the original, but requires less typing and ties the
+layout of the code to the way that code is interpreted.
```

Comment:
My thought it that it would have to entirely indentation sensitive to distinguish environment from non-environment.

I could be wrong though - I haven't thought this entirely through.